core/src/main/java/org/elasticsearch/search/aggregations/bucket/significant/SignificantTermsAggregatorFactory.java

```diff
@@ -293,7 +294,7 @@ public static ExecutionMode fromString(String value) {
                     return mode;
                 }
             }
-            throw new IllegalArgumentException("Unknown `execution_hint`: [" + value + "], expected any of " + values());
+            throw new IllegalArgumentException("Unknown `execution_hint`: [" + value + "], expected any of " + Arrays.toString(values()));
```

Comment:
Looking at the blame for the file: https://github.com/elastic/elasticsearch/blame/ab11be55a2276e66226cd84b0de65d0696a4fb94/src/main/java/org/elasticsearch/search/aggregations/bucket/significant/SignificantTermsAggregatorFactory.java#L103

It's been there since April 29 2014, added by martijnvg, holding back on doing an actual ping.

Wild stab in the dark: It's simply for drawing attention to the term when someone reads it, agnostic of markdown existing.